README.adoc

```diff
@@ -56,38 +56,38 @@ include::{common-includes}/gitclone.adoc[]
 
 The `finish` directory in the root of this guide contains the finished application. Give it a try before you proceed.
 
-First, look at [hotspot file=0]`PersonServiceIT` class to see what the tests look like:
+First, review the [hotspot file=0]`PersonServiceIT` class to see what the tests look like:
 
 PersonServiceIT.java
 [source, java, linenums, role='code_column hide_tags=copyright']
 ----
 include::finish/src/test/java/io/openliberty/guides/testing/PersonServiceIT.java[]
 ----
 
-To try out the application, first go to the `finish` directory and run the following Maven 
+To try out the application, go to the `finish` directory and run the following Maven 
 goal to build the application and run the integration tests on an Open Liberty server in a container:
 [role='command']
 ```
 mvn verify
 ```
 
-This may take some time on the first run, due to downloading dependencies and the Docker image for Open Liberty. If you 
-rerun the same command again, it will be faster.
+This command might take some time to run the first time because the dependencies and the Docker image for Open Liberty must download. If you 
+run the same command again, it is faster.
```

Comment:
I think `will be` make more sense here because we are talking about something the user might do at some point in the future

Reply:
@aguibert Okay. So, `This command might take some time to run the first time because the dependencies and the Docker image for Open Liberty must download. If you 
run the same command again, it will be faster.`?

Reply:
@Charlotte-Holt what do you think?

Reply:
@aguibert is it ^ OK?

Reply:
correct